icechunk-python/python/icechunk/repository.py

```diff
@@ -564,6 +566,62 @@ def writable_session(self, branch: str) -> Session:
         """
         return Session(self._repository.writable_session(branch))
 
+    @contextmanager
+    def transaction(self, branch: str, message: str) -> Iterator["IcechunkStore"]:
```

Comment:
```suggestion
    def transaction(self, branch: str, *, message: str) -> Iterator["IcechunkStore"]:
```

mild preference for making this explicit